memo/program-id.md

```diff
@@ -0,0 +1 @@
+Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo
```

Comment:
for the convenience of Rust clients, maybe the program-id should live in a .rs file too?  I guess this is a more general SPL setup comment that we can deal with later

Reply:
Yeah, good idea. I'll leave as-is for now, but will move around when we rethink how we want to publish rust assets.